RoosterPlanner.UI/src/app/models/shift.ts

```diff
@@ -0,0 +1,12 @@
+﻿import {Entity} from "./entity.model";
+import {Task} from './task';
+import {Project} from "./project";
+
+export class Shift extends Entity {
```

Comment:
Geen initiele waarden?

Reply:
Wat zou er initieel ingevuld moeten worden?

Reply:
Is meer een trigger, misschien niks..

Reply:
Duidelijk. Niet mijn voorkeur in front-end modellen maar anders zou ik verplichte parameters opgeven in de constructor